main/common/system-permissions.js

```diff
@@ -11,19 +10,31 @@ const promptSystemPreferences = async ({hasAsked} = {}) => {
     type: 'warning',
     buttons: ['Open System Preferences', 'Cancel'],
     defaultId: 0,
-    message: 'Kap cannot access the microphone.',
-    detail: 'Kap requires microphone access to be able to record audio. You can grant this in the System Preferences. Afterwards, relaunch Kap for the changes to take effect.',
+    message: opts.message,
+    detail: opts.detail,
     cancelId: 1
   });
 
   if (response === 0) {
-    openSystemPreferences();
+    openSystemPreferences(opts.systemPreferencesPath);
   }
 
   return false;
 };
 
-const ensureMicrophonePermissions = async (fallback = promptSystemPreferences) => {
+const openSystemPreferences = path => shell.openExternal(`x-apple.systempreferences:com.apple.preference.security?${path}`);
```

Comment:
Your `mac-screen-capture-permissions` should ideally provide a method to open the correct preferences pane.

Ideally, `electron-util` would have a utility method to open system preferences, which could be used in your package.

Reply:
I'll open a PR to add it to electron-util, you're right it should live there

Reply:
Opened a PR: https://github.com/sindresorhus/electron-util/pull/23
After that is merged and released, I can make the updates to the package, then release and then I can update the dependency here

We could keep it as is, until that is done, or if you have some time to review that PR, I can update and bump the dependency in this PR